packages/nimble-components/src/toggle-button/tests/toggle-button.mdx

```diff
@@ -0,0 +1,25 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import * as toggleButtonStories from './toggle-button.stories';
+
+<Meta of={toggleButtonStories} />
+<Title of={toggleButtonStories} />
+<Description of={toggleButtonStories} />
+
+<DocsStory of={toggleButtonStories.outlineButton} expanded={false} />
+<Controls of={toggleButtonStories.outlineButton} />
+<Stories of={toggleButtonStories} />
+
+# Usage Docs
+
+## Accessibility
+
+When using the button with only an icon, `contentHidden` should be set to `true` and text content should be provided within the button
+even though it will not be visible on the screen. The button will use the text content to configure the appropriate ARIA attributes internally
+to ensure the button is adequately accessible with a screen reader.
```

Comment:
mdx files are components, so we can create re-usable mdx components for these shared docs
See "MDX files are components": https://mdxjs.com/docs/using-mdx/#components

Reply:
Split this text out into `patterns/button/tests/content-hidden-docs.mdx` and referenced it as a component in the button mdx files.